prow/github/fakegithub/fakegithub.go

```diff
@@ -512,12 +513,12 @@ func (f *FakeClient) DeleteProjectCard(projectCardID int) error {
 	return nil
 }
 
-func (f *FakeClient) GetColumnProjectCard(columnID int, cardNumber int) (*github.ProjectCard, error) {
+func (f *FakeClient) GetColumnProjectCard(columnID int, contentURL string) (*github.ProjectCard, error) {
 	if f.ColumnCardsMap == nil {
 		f.ColumnCardsMap = make(map[int][]github.ProjectCard)
 	}
 	for _, existingCard := range f.ColumnCardsMap[columnID] {
-		if existingCard.ContentID == cardNumber {
+		if strings.Compare(existingCard.ContentURL, contentURL) == 0 {
```

Comment:
This re-implements the logic in the actual client and does it differently?

Reply:
I now has same logic.

Reply:
I think the point being that if we're implementing logic in the test fake we're not testing the actual code, this makes me wonder if we could? Maybe share a helper?

Reply:
I agree, I added a helper to match card to content url of the issue/pr. we now can match the issue/pr using both url and html_url fields.